src/Samples/Toolkit.SampleApp.Maui/Resources/Styles/Colors.xaml

```diff
@@ -17,8 +17,8 @@
     <Color x:Key="Gray600">#404040</Color>
     <Color x:Key="Gray900">#212121</Color>
     <Color x:Key="Gray950">#141414</Color>
-    <SolidColorBrush x:Key="PrimaryBrush" Color="{StaticResource Primary}"/>
-    <SolidColorBrush x:Key="SecondaryBrush" Color="{StaticResource Secondary}"/>
+    <SolidColorBrush x:Key="PrimaryBrush" Color="{AppThemeBinding Dark={StaticResource Tertiary}, Light={StaticResource Primary}}"/>
+    <SolidColorBrush x:Key="SecondaryBrush" Color="{AppThemeBinding Dark={StaticResource Tertiary}, Light={StaticResource Primary}}"/>
```

Comment:
Isn't it better to declare the Primary color by apptheme instead so anything using 'Primary" color will use it?

Reply:
Maybe, but that would require that the value for the color is a bindable property, and I don't think it is.